### ISSUE TITLE
[@fab/nextjs] Expects static directory in project root where none exists

### ISSUE THREAD

**Issue by plexey** [fab-spec/fab#50]
Following the [FAB docs](https://fab.dev/packages/fab-nextjs), I configured a new Nextjs 9 project that I generated with `create-next-app` to output a FAB using version `0.4.0` of `@fab/nextjs`.

When I ran `yarn build:fab`, I received the following error message:
```bash
[FAB:NextJS] Copying static files
Error: ENOENT: no such file or directory, stat '/home/plexey/work/nextjs9-demo/static'
```
It appears that `@fab/nextjs` expects a `static` directory to exist at the root of your Nextjs project where none exists, at least in the case of a project generated by `create-next-app`. I think the line in question is here: https://github.com/fab-spec/fab/blob/next/old-packages/fab-nextjs/src/Builder.ts#L45

After adding an empty `static` directory to the root of my project, `@fab/nextjs` was then able to successfully compile a FAB. Serving the FAB locally using `npx @fab/serve fab.zip` also worked successfully.

**Comment by geelen** [comment]
Published `@fab/nextjs@0.4.1` which should fix this in the interim while I work on v1

**Comment by geelen** [comment]
Fixed in v1
